spec/app.spec.js

```diff
@@ -5,7 +5,7 @@ describe('app.spec: app namespace tests', () => {
 
     describe('app.exit', () => {
         it('works without parameters', async () => {
-            let exitCode = runner.run(`
+            let exitCode = runner.runInBackground(`
```

Comment:
Hey, `runner` does not expose any function called `runInBackground` If you have added that function please ensure that you have pushed that code because I don't see any changes made to `runner.js`